Platon/backend/app/workspace_system/views.py

```diff
@@ -1510,6 +1563,32 @@ def delete(invitee_id, self):
                             contribution = Contribution(workspace_id=invitation.workspace_id,user_id=invitee_id,is_active=True)
                             db.session.add(contribution)
                             add_new_collaboration(invitation.workspace_id, invitee_id)
+    
+                            try:
+                                workspace = Workspace.query.filter(Workspace.id == invitation.workspace_id).first()
+                                current_user = User.query.filter(User.id ==invitee_id).first()
+                            except:
+                                return make_response(jsonify({"error" : "The server is not connected to the database."}), 500)
+
+                            # Activity is added into Activity Stream
+                            activity_stream_entry = ActivityStreamItem(
+                                activity_context_vocab = "https://www.w3.org/ns/activitystreams",
+                                activity_summary = "{} {} is now a contributor in {} workspace".format(current_user.name, current_user.surname, workspace.title),
+                                activity_type = "Join",
+                                activity_actor_type = "Person",
+                                activity_actor_id = current_user.id,
+                                activity_actor_name = (current_user.name + " " + current_user.surname),
+                                activity_actor_image_url = profile_photo_link(current_user.profile_photo,current_user.id),
+                                activity_object_type = "Group",
+                                activity_object_name = workspace.title,
+                                activity_object_id = workspace.id,
+                            )
+                            try:
+                                db.session.add(activity_stream_entry)
+                                # commit is done 5 lines below.
+                            except:
+                                return make_response(jsonify({'error': 'Database Connection Error'}), 500)
+                            
```

Comment:
A small comment on these parts: I think it will be better to implement these parts as some helper functions of the activity stream. Because we know that previous parts are working correctly, it will be easier to find bugs for us.